packages/@aws-cdk/aws-ecs/lib/container-definition.ts

```diff
@@ -38,6 +38,81 @@ export abstract class Secret {
   public abstract grantRead(grantee: iam.IGrantable): iam.Grant;
 }
 
+/**
+ * Firelens log router type
+ * https://docs.aws.amazon.com/AmazonECS/latest/developerguide/using_firelens.html
+ */
+export enum FireLensRouterType {
```

Comment:
Can we rename this to FirelensLogRouterType?

Reply:
sure, renamed in the latest commit: https://github.com/aws/aws-cdk/pull/5457/files#diff-541fb20ec37d9e651dd38abc923ccb58R45